website/README.md

```diff
@@ -28,7 +28,8 @@ sudo npm i -g yarn
 ### Make sure you have the output from the Go program
 
 To run the website, you must follow the steps in `running code` in the primary README.\
-The `output` folder in the root of the repository must be present.
+The `output` folder in the root of the repository should be present if you do this successfully.
```

Comment:
following #202 this is not needed -- `yarn start` does this automatically and even runs the sim if it's not ran.

Reply:
have removed in latest push